src/app/collection/pages/collection-502/collection-502.component.scss

```diff
@@ -29,11 +29,13 @@
     margin-left: 10px;
   }
 }
-::ng-deep .mat-slide-toggle.mat-checked .mat-slide-toggle-bar {
+/* TODO(mdc-migration): The following rule targets internal classes of slide-toggle that may no longer apply for the MDC version. */
```

Comment:
The documentation mentions we will have to verify these are still working.
`TODO(mdc-migration)`